pyanaconda/modules/storage/kickstart.py

```diff
@@ -72,6 +72,7 @@ class BTRFS(COMMANDS.BTRFS):
 
     def parse(self, args):
         """Parse the command."""
+        # pylint: disable=assignment-from-no-return
```

Comment:
The upstream should be fixed with https://github.com/pykickstart/pykickstart/pull/386.

Reply:
Thanks! :)